packages/components/src/components/PipelineRun/PipelineRun.js

```diff
@@ -47,6 +47,15 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
 
   loadPipelineRunData = () => {
     const { pipelineRun } = this.props;
+
+    if (!pipelineRun.status) {
+      return {};
+    }
+
+    if (!pipelineRun.status.taskRuns) {
```

Comment:
let's combine these:

```js
if (!(pipelineRun.status && pipelineRun.status.taskRuns)) {
```

can we return more than just an empty object? e.g. at least `{ pipelineRun }`